<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataOptionsPage</class>
 <widget class="QWidget" name="MetadataOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>553</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="metadata_groupbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>397</width>
       <height>135</height>
      </size>
     </property>
     <property name="title">
      <string>Metadata</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="translate_artist_names">
        <property name="text">
         <string>Translate artist names to these locales where possible:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="selected_locales">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_locales">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="translate_artist_names_script_exception">
        <property name="text">
         <string>Ignore artist name translation for these scripts:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ignore_script_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListWidget" name="ignore_tx_scripts">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Only scripts where the percentage of the artist name that is written using that script are above the specified minimum weighting will be considered.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Minimum Weighting:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="minimum_weighting">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="standardize_artists">
        <property name="text">
         <string>Use standardized artist names</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="standardize_instruments">
        <property name="text">
         <string>Use standardized instrument and vocal credits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="convert_punctuation">
        <property name="text">
         <string>Convert Unicode punctuation characters to ASCII</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="release_ars">
        <property name="text">
         <string>Use release relationships</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="track_ars">
        <property name="text">
         <string>Use track relationships</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="guess_tracknumber_and_title">
        <property name="text">
         <string>Guess track number and title from filename if empty</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="custom_fields_groupbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>397</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Custom Fields</string>
     </property>
     <layout class="QGridLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Various artists:</string>
        </property>
        <property name="buddy">
         <cstring>va_name_default</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Non-album tracks:</string>
        </property>
        <property name="buddy">
         <cstring>nat_name_default</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="nat_name"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="nat_name_default">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="va_name_default">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="va_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>translate_artist_names</tabstop>
  <tabstop>translate_artist_names_script_exception</tabstop>
  <tabstop>standardize_artists</tabstop>
  <tabstop>standardize_instruments</tabstop>
  <tabstop>convert_punctuation</tabstop>
  <tabstop>release_ars</tabstop>
  <tabstop>track_ars</tabstop>
  <tabstop>guess_tracknumber_and_title</tabstop>
  <tabstop>va_name</tabstop>
  <tabstop>va_name_default</tabstop>
  <tabstop>nat_name</tabstop>
  <tabstop>nat_name_default</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
